### PR TITLE
Add steps for running operator e2e tests

### PR DESCRIPTION
**Pull request by gnufied**
Requires changes proposed in https://github.com/openshift/vmware-vsphere-csi-driver-operator/pull/248

### CODE REVIEW

**Comment by gnufied** [comment]
/pj-rehearse pull-ci-openshift-vmware-vsphere-csi-driver-operator-master-operator-e2e-test

File: ci-operator/config/openshift/vmware-vsphere-csi-driver-operator/openshift-vmware-vsphere-csi-driver-operator-master.yaml [hunk 1]
```diff
@@ -63,6 +63,18 @@ tests:
       TEST_SKIPS: provisioning should mount multiple PV pointing to the same storage
         on the same node
     workflow: openshift-e2e-vsphere
+- as: operator-e2e-test
```
Comment (jsafrane) [inline ci-operator/config/openshift/vmware-vsphere-csi-driver-operator/openshift-vmware-vsphere-csi-driver-operator-master.yaml:66]: looking at the other e2e jobs: `e2e-vsphere`, `e2e-vsphere-csi`, `e2e-vsphere-csi-extended`, `e2e-vsphere-zones`, should it be `e2e-vsphere-operator`?

**Comment by gnufied** [comment]
/pj-rehearse pull-ci-openshift-vmware-vsphere-csi-driver-operator-master-operator-e2e-test

**Comment by gnufied** [comment]
/pj-rehearse pull-ci-openshift-vmware-vsphere-csi-driver-operator-master-e2e-vsphere-operator-test

**Comment by gnufied** [comment]
/pj-rehearse pull-ci-openshift-vmware-vsphere-csi-driver-operator-master-e2e-vsphere-operator-test

**Pull request merged**